plugin/src/main/java/net/dzikoysk/funnyguilds/guild/permission/GuildPermissionController.java

```diff
@@ -0,0 +1,71 @@
+package net.dzikoysk.funnyguilds.guild.permission;
+
+import net.dzikoysk.funnyguilds.guild.Guild;
+import net.dzikoysk.funnyguilds.user.User;
+import org.bukkit.event.Event;
+import panda.std.Option;
+import panda.std.Result;
+
+public interface GuildPermissionController {
```

Comment:
Jak widzę w tej nazwie controller to dostaję ataków paniki, daj po prostu `GuildPermissionChecker`